assets/js/modules/tagmanager/datastore/settings.test.js

```diff
@@ -342,6 +391,12 @@ describe( 'modules/tagmanager settings', () => {
 		} );
 
 		describe( 'canSubmitChanges', () => {
```

Comment:
Please add cases here for ensuring that a valid container name is required for each AMP mode.

Reply:
It's already added there or you want to add something else?

Reply:
Hmm, I see the cases where we test for a _unique_ container name, but not where we test for a valid container name which is different (i.e. checks for invalid characters). If it is there and I'm not seeing it, please point me to it! 😄

Reply:
Ok, added a few tests that check cases when a container name is invalid.